tests/lbs/main.c

```diff
@@ -296,13 +298,13 @@ main(int argc, char * argv[])
 	}
 
 	/* Attempt to read a non-existent block. */
-	get_done = get_failed = 0;
+	get_done = get_not_exist_response = 0;
 	if (proto_lbs_request_get(Q, BAD_BLKNO, params_blklen,
 	    callback_get_should_not_exist, NULL)) {
 		warnp("Failed to send GET request");
 		goto err2;
 	}
-	if (events_spin(&get_done) || get_failed) {
+	if (events_spin(&get_done) || get_not_exist_response) {
```

Comment:
Hmm, I was thinking that this would end up as

```
if (events_spin(&get_done) || get_failed) {
    warnp("GET request failed");
    goto err2;
}
if (get_not_exist_response != 1) {
    warnp("GET request failed to return does-not-exist");
    goto err2;
}
```

so that the warning message printed would indicate what broke.

Reply:
oh, I see!  Sure, that's do-able.

I'd be shocked if the previous GET request tests passed, but this one failed (in the non-`not_exist_response` sense), but there's no harm in testing for that, too.

Reply:
I would be surprised but not shocked -- say if there was a bug which made lbs crash when it tried to send a "does not exist" response.

Reply:
ok, done with REBASE commits.